### PR TITLE
feat: Security Page

### PR DESCRIPTION
**Pull request by natapg**
Closes [WEB-433](https://fpjs.myjetbrains.com/youtrack/issue/WEB-433)
https://www.figma.com/file/niHrIRowTbqQhDO7W5CPNf/Web-Mockups?node-id=5663%3A45269

### CODE REVIEW

File: src/pages/security/Security.module.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  align-items: center;
+
+  margin-bottom: rem(96px);
+  background: linear-gradient(95.53deg, #f25250 -1.57%, #fa7247 102.59%);
+
+  @include media('<=phoneLandscape') {
+    padding: rem(40px);
+  }
+
+  //TODO create new banner variant
```
Comment (tan-90) [inline src/pages/security/Security.module.scss:16]: Do we still need it?

**Review by tan-90 — APPROVED** [review]
Please take a look at the comments but they're all minor, approving in advance.
The page looks great!

File: src/components/security/HeroSection/HeroSection.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import Section from '../../common/Section'
+import Container from '../../common/Container'
+import ShieldSVG from '-!svg-react-loader!./images/shield.inline.svg'
```
[the remark this answers is not in the document]
Comment (natapg) [reply src/components/security/HeroSection/HeroSection.tsx:5]: yes, svgr is removing the script even disabling the svgo option, using this loader does not delete it

File: src/pages/security/Security.module.scss (hunk 1 above)
Comment (natapg) [reply src/pages/security/Security.module.scss:16]: nope, I'll remove it

**Pull request merged**